test/e2e/framework/util.go

```diff
@@ -1668,7 +1668,7 @@ func DeleteResourceAndWaitForGC(c clientset.Interface, kind schema.GroupKind, ns
 		timeout = timeout + 3*time.Minute
 	}
 
-	err = e2epod.WaitForPodsInactive(ps, interval, timeout)
+	err = waitForPodsInactive(ps, interval, timeout)
```

Comment:
This PR brings up an interesting issue.
The function that is calling `e2epod` is `DeleteResourceAndWaitForGC()`.
The previous consensus was to not call framework subpackages withing the framework (i.e., avoid e2epod calls inside of the framework) but the other driving interest of the project was to clean up the test framework and separate all different behaviors into subpackages.
/hold

Reply:
Additional comments  https://github.com/kubernetes/kubernetes/pull/83814#discussion_r334217997

Reply:
Emm, I'm a little confused, too. :confused: 
We are trying to break up `framework/utils.go`, right?
Ref: https://github.com/kubernetes/kubernetes/issues/77095

Reply:
For extra reference: here is the place within the framework where the method is called https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/util.go#L1619